scripts/utils/keeper_registry.ts

```diff
@@ -0,0 +1,175 @@
+import { BigNumber, Wallet, constants } from "ethers";
+import { ethers } from "hardhat";
+import { KeeperRegistry20 } from "../../typechain";
+
+const { HashZero } = ethers.constants;
+
+const linkEth = BigNumber.from(5000000000000000); // 1 Link = 0.005 Eth
+const gasWei = BigNumber.from(1000000000); // 1 gwei
+const EPOCH_AND_ROUND_5_1 =
+  "0x0000000000000000000000000000000000000000000000000000000000000501";
+
+function encodeReport(
```

Comment:
Aren't these functions available in the `@chainlink/contracts` package?

Reply:
Cant seem to find them in this package. All I can see in this package is `.sol` files and abi files.